app/controllers/application_controller.rb

```diff
@@ -1,16 +1,19 @@
 class ApplicationController < ActionController::Base
-  before_action :authenticate_user!
+  # before_action :authenticate_user!
 
   def after_sign_in_path_for(resource)
     # debugger
+    default_user_path(resource)
+  end
+
+  def default_user_path(resource)
     if resource.has_role?('project_manager')
-        managers_path
+      managers_path
     elsif resource.has_role?('accountant')
-        accountants_path
+      accountants_path
     else
-        employees_path
-      end
-
+      employees_path
     end
```

Comment:
why we need a separate function for this redirection use this code in the same function after_sign_in_path

Reply:
remove the separate function